src/libraries/Stack.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+contract Stack {
+    
+    var[] collection;
+    uint head;
+    address owner;
+    bool initialized;
+
+    function any() {
+         if (initialized) {
+             require(msg.sender == this.owner || msg.sender == this);
+         }
+    }
+
+    function initialize(address owner) returns (uint) {
+         require(!this.initialized);
+         this.initialized = true;
+         this.owner = owner;
+         return (uint(1));
+    }
+
+    function push(var item) returns (uint) {
```

Comment:
Does this actually work (var parameter)?  I would be quite surprised if it did...